extension/intents/routines/routines.js

```diff
@@ -98,7 +98,13 @@ intentRunner.registerIntent({
   name: "routines.removePageName",
   async run(context) {
     const name = context.slots.name;
-    await intentRunner.getRegisteredPageName(name);
+    const pageNames = await intentRunner.getRegisteredPageName(name);
```

Comment:
Here `getRegisteredPageName()` is used with an argument, but the function as defined doesn't take any arguments.

Really there's a confusion here, as `getRegisteredPageName()` is _also_ used elsewhere without an argument (in `registerPageName()`). But in that case it's being used incorrectly.

Be sure when you are changing a function that you look for everywhere where the function is used, to make sure you are changing it in a way that's compatible with how it's being used.